ProjectSettings/ProjectSettings.asset

```diff
@@ -46,7 +46,7 @@ PlayerSettings:
   defaultScreenHeight: 768
   defaultScreenWidthWeb: 960
   defaultScreenHeightWeb: 600
-  m_StereoRenderingPath: 0
+  m_StereoRenderingPath: 2
```

Comment:
@Cameron-Micka I hadn't noticed we changed this. Any chance we could put it back to multipass?

The Single Pass Instanced is still in preview

Reply:
I can certainly switch it back, or move to normal Single Pass? My team has been using Single Pass Instanced for all of our projects (even enterprise applications delivered to clients) and it has been super stable and a big perf win.

I think the “preview” portion is a handful of Unity extensions haven’t been updated to utilize it. Such as TextMesh Pro. But, it’s pretty easy to fix those instances. (No pun intended. 😉)

Reply:
I'm getting a slight stutter every few seconds with single pass enabled.

Reply:
Ah interesting, are you seeing it in a particular scene or all scenes? I suppose we have only been using Unity 2017 LTS so haven’t done much testing in 2018.

Reply:
It looks like #2521 might change this setting? But, please let me know if there is anything you need me to do.

Reply:
Yup, I had requested @keveleigh to change it back. Thanks 👍